apps/coin/contracts/Coin.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.4.24;
+
+import "@aragon/os/contracts/apps/AragonApp.sol";
+import "@aragon/os/contracts/lib/math/SafeMath.sol";
+
+import "./StandardCoin.sol";
+
+contract Coin is StandardCoin, AragonApp {
+    
+    using SafeMath for uint256;
+
+    bytes32 public constant ISSUE_ROLE = keccak256("ISSUE_ROLE");
+    bytes32 public constant MINT_ROLE = keccak256("MINT_ROLE");
+    //bytes32 public constant TRANSFER_ROLE = keccak256("TRANSFER_ROLE");
+    //bytes32 public constant ASSIGN_ROLE = keccak256("ASSIGN_ROLE");
+
+    string private _name;
+    string private _symbol;
+    uint8 private _decimals;
+
+    function initialize() public onlyInit {
+
+        initialized();
+    }
+
+    function name() public view isInitialized returns(string) {
+        return _name;
+    }
+
+    function symbol() public view isInitialized returns(string) {
+        return _symbol;
+    }
+
+    function decimal() public view isInitialized returns(uint8) {
+        return _decimals;
+    }
+
+    function issueToken(string memory name, string memory symbol, uint8 decimals) public isInitialized auth(ISSUE_ROLE) {
+        _name = name;
+        _symbol = symbol;
+        _decimals = decimals;
+    }
+
+    function mintToken(uint256 amount) public isInitialized auth(MINT_ROLE) {
+        require(amount > 0, "amount equal to zero");
```

Comment:
I am wondering if we should do a mint function that can set a recipient of the coins. 
This would make it easier to track the amount of coins that are issued for certain reasons to an account and it would not need to go through the manager account.

Reply:
I didn't understand that very well... if u can explain more please

Reply:
thinking about that [contribution/issuance contract](https://drive.google.com/drive/u/2/folders/1dbGffo11j_1tkRsYcwY0-SCmEebwgBbo) that allows us to add metadata to the issuance (for example why somebody gets those coins) and that contract could issue the tokens to the member (similar to what we do in kredits). 

in anyway a `mintFor(recipient, amount)` function would allow us to directly mint coins to any account and not only the msg.sender. 
e.g. if I am the space admin, and I want to issue coins for you, now I would need to mint tokens and then transfer them to you. 
like that we also don't see if that was a normal transfer (me just selling/giving coins to you) or if those were issued for you for a specific task/contribution that you did.